src/frontend/components/node-item/node-attributes.html

```diff
@@ -1,5 +1,5 @@
 <span *ngIf="attributes?.length > 0">
   <span class="node-item-description">
-    (<template ngFor let-d [ngForOf]="attributes" let-last="last"><p class="node-item-property">{{d.key}}</p><span class="punctuation">=&quot;</span><p class="node-item-value">{{d.value}}</p><span class="punctuation">&quot;</span><span *ngIf="!last" class="punctuation">,&nbsp;</span></template>)
+    <!-- (<template ngFor let-d [ngForOf]="attributes" let-last="last"><p class="node-item-property">{{d.key}}</p><span class="punctuation">=&quot;</span><p class="node-item-value">{{d.value}}</p><span class="punctuation">&quot;</span><span *ngIf="!last" class="punctuation">,&nbsp;</span></template>) -->
```

Comment:
is this meant to be commented out?

Reply:
@sumitarora can you take a look? This was commented out and we just copied these changes and nothing broke as far as I can tell.